### ISSUE TITLE
metal: Devise release key management

### ISSUE THREAD

**Issue by kim** [oscoin/ci#64]
All releases should be cut from the #63 box, and appropriately signed.

[signify](https://www.openbsd.org/papers/bsdcan-signify.html) is a good tool to avoid GPG and the associated key management headaches, but GPG may still be needed for some use cases (e.g. `deb` packages).

Access to storage platforms for the produced artifacts also needs to be provided for the #63 box, with appropriate documentation and credential rotation procedures. This is part of this issue.

Depends on #63
